### PR TITLE
New plugin Color Chooser, no.2

### PR DESCRIPTION
**Pull request by palinek**
Reviving the #939

### CODE REVIEW

**Comment by stefonarch** [comment]
Works fine, but the color icon is some strange, shouldn't it be a simple circle?

schermata-03-11-14-44-01

**Comment by palinek** [comment]
> Edit: it looks like another drop is behind it.

That's a theme stylesheet clash, e.g. https://github.com/lxqt/lxqt-themes/blob/master/themes/frost/lxqt-panel.qss#L659

**Comment by palinek** [comment]
The whole Color Picker specific styling should be removed from themes.

**Comment by stefonarch** [comment]
> The whole Color Picker specific styling should be removed from themes.

I see, removing it it's ok. Only that I had problems to understand the icon, it looks like too much like a sword:


schermata-03-11-15-24

**Comment by stefonarch** [comment]
I can't speak for the code,  but it works well. GTM

File: plugin-colorpicker/colorpicker.cpp [hunk 1]
```diff
@@ -44,37 +71,76 @@ ColorPicker::~ColorPicker() = default;
...
+    mSeparator->setFrameShape(QFrame::VLine);
+    mSeparator->setFrameShadow(QFrame::Sunken);
+    mSeparator->setLineWidth(1);
+    mSeparator->setFixedHeight(16);
 
+    mPickerButton = new QToolButton();
+    mPickerButton->setObjectName(QStringLiteral("ColorPickerPickerButton"));
+    mPickerButton->setAccessibleName(mPickerButton->objectName());
+    mPickerButton->setAutoRaise(true);
+    mPickerButton->setIcon(QIcon::fromTheme(QLatin1String("color-select-symbolic"), QIcon::fromTheme(QLatin1String("color-picker"))));
```
Comment (tsujan) [inline plugin-colorpicker/colorpicker.cpp:92]: Although an icon set like Breeze has "xyz-symbolic" icons, they're mainly for Gnome. IMHO, it's better to reverse the fallback and main icons here.

Comment (palinek) [reply plugin-colorpicker/colorpicker.cpp]: Done

**Review by tsujan — APPROVED** [review]
GTM

**Pull request merged**